src/components/accordion/accordion.component.tsx

```diff
@@ -21,7 +21,7 @@ export interface AccordionProps
   extends StyledAccordionContainerProps,
     SpaceProps {
   /** Width of the buttonHeading when it's set, defaults to 150px */
-  buttonWidth?: number;
+  buttonWidth?: string;
```

Comment:
I'm wondering if we should avoid the breaking change here and support both `number` and `string`. If it's a number value then we can keep the current behaviour of appending `px` to it. If it's a string we just apply the value and let the implementing developers set the unit etc. What do you think?